src/main/java/org/myrobotlab/service/config/ImageDisplayConfig.java

```diff
@@ -0,0 +1,59 @@
+package org.myrobotlab.service.config;
+
+import java.util.HashMap;
+
+import javax.swing.JFrame;
+
+import org.myrobotlab.math.geometry.Rectangle;
+
+public class ImageDisplayConfig extends ServiceConfig {
+
+  
+    public static class Display{
+      // problematic - make transient
+      String name;
+      
+      public Integer x, y;
+      public Integer width, height;
+      
+      public Boolean fullscreen = null;
```

Comment:
this should have a default value.. we should avoid nulls as defaults.

Reply:
config potentially can have 3 values - the semantics are simple
true config force true behavior
false config force false behavior
null config do not influence current behavior

Reply:
so long as getConfig doesn't provide any nulls in the config and fully populates all values with non null values , i'm ok with it.

Reply:
updated to boolean, Integer x, y, width height as null means in the center of the screen as the actual image size